c7n/resources/keyspaces.py

```diff
@@ -0,0 +1,303 @@
+from c7n.actions import BaseAction
+from c7n.manager import resources
+from c7n.query import ChildResourceManager, DescribeSource, QueryResourceManager, TypeInfo
+from c7n.resources.aws import shape_schema
+from c7n.tags import RemoveTag, Tag, TagActionFilter, TagDelayedAction
+from c7n.utils import get_retry, local_session, type_schema
+
+
+SYSTEM_KEYSPACES = [
+    "system",
+    "system_schema",
+    "system_schema_mcs",
+    "system_multiregion_info",
+]
+
```

Comment:
In a recent community meeting there was some discussion about using pattern matching here instead of a hardcoded list. I can see that AWS lets you create keyspaces like `system_meh` though, so the hardcoded list seems like the way to go :+1: